pl_examples/basic_examples/backbone_image_classifier.py

```diff
@@ -66,11 +66,13 @@ class LitClassifier(pl.LightningModule):
 
     def __init__(
         self,
-        backbone,
+        backbone: Backbone = None,
```

Comment:
Would this work too?

```suggestion
        backbone: Backbone = Backbone(),
```

Reply:
There are some problems with that:
- `backbone` would be instantiated on load of `backbone_image_classifier.py`, which means it would happen before `seed_everything`.
- `--print_config` does not work because there is no general way to convert an arbitrary instance of a class, like `Backbone()` to `{"class_path": "pl_examples.basic_examples.backbone_image_classifier.Backbone", "init_args": {}}`.

For me the normal case should be have no default, thus being required. But then the current way to call this example by people checking it out or in the tests wouldn't work.

Reply:
Okay! Let's keep it as it is